py2exe/hooks.py

```diff
@@ -805,3 +869,41 @@ def hook_wx(finder, module):
     when importing the full wx package
     """
     finder.excludes.append("wx.lib.pubsub")
+
+
+def hook_zmq(finder, module):
+    """pyzmq  >= 3.7.0 moved .dlls to a matplotlib.libs directory,
```

Comment:
The minimum version does not match what is used in the code, please update the docstring accordingly.